manifests/forward.pp

```diff
@@ -21,12 +21,26 @@
 #     systems where you want a minimum of log information to be captured
 #     locally.
 #
+# @param permitted_peers
+#   If TLS is being used, StreamDriverPermittedPeers is used to check the certificates.
+#   permitted_peers is used to set this.  In general, certificates will use the FQDN
+#   of the host as the CN in the subject of the certificate.  The default, "LOGSERVERNAME"  assumes that
+#   $::simp_rsyslog::log_servers and  $::simp_rsyslog::failover_log_servers are a list
+#   of FQDNs and that certificates used by the log servers to encrypt traffic use this for the CN.
+#   If this is not the case, then you will need to set this variable appropriately.
+#   Example:  If IP Addresses are used in the two simp_rsyslog settings above and the CN in 
+#   the servers certs are server.my.domain and server2,my.other.domain: 
+#   Set this to "server.my.domain,server2.my.other.domain"
+#   @see https://www.rsyslog.com/doc/v8-stable/configuration/modules/omfwd.html
+#   for more information on how to set this.
+#
 # @author Trevor Vaughan <tvaughan@onyxpoint.com>
 #
 class simp_rsyslog::forward (
-  Integer                  $order           = 99,
-  Enum['tcp','udp','relp'] $dest_type       = 'tcp',
-  Boolean                  $stop_processing = false
+  Integer                  $order            = 99,
+  Enum['tcp','udp','relp'] $dest_type        = 'tcp',
+  Boolean                  $stop_processing  = false,
+  String                   $permitted_peers  = 'LOGSERVERNAME'
```

Comment:
Shouldn't this default to $::simp_rsyslog::log_servers + $::simp_rsyslog::failover_log_servers